src/richie/apps/core/templates/djangocms_video/default/video_player.html

```diff
@@ -0,0 +1,33 @@
+{% load i18n cms_tags %}
+{% comment %}
+This is a copy of original template from plugin just to clean <iframe> from
+obsolete attribute "frameborder" and invalid "allowfullscreen" attribute value.
+{% endcomment %}
+
+{% if instance.embed_link %}
+    {# show iframe if embed_link is provided #}
+    <iframe src="{{ instance.embed_link_with_parameters }}" {{ instance.attributes_str }} allowfullscreen></iframe>
```

Comment:
@sveetch In your commit message you say that allowfullscreen is invalid but you are keeping it here?

Reply:
@sampaccoud it does not seems so, i thinks it's typo error from me in my commit message. But finally i checked from MDN and it does seems it could be replaced by `allow="fullscreen"`  because `allowfullscreen` is not deprecated but marked as "historical attribute".

Reply:
I propose to keep `allowfullscreen`.